Styles.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
+body {
+  margin-right: 486.5px;
+  margin-left: 486.5px;
+}
```

Comment:
- [ ] Kindly make sure your web page can fit the mobile device.
How is it possible a mobile that has a width of 474-320px can contain very well a page where the body is 486.5px margin-left and right?
This is how your page looks like
![Screenshot from 2022-09-23 09-05-46](https://user-images.githubusercontent.com/97100091/191909409-d98710fd-a783-43c0-8fde-8a74d49692ee.png)
 - [ ] Kindly make sure you attach as much as possible to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design provided by Microverse